hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/KeyValueContainerUtil.java

```diff
@@ -197,15 +197,19 @@ public static boolean noBlocksInContainer(DatanodeStore store,
    * pending delete block count and delete transaction id.
    * @param kvContainerData
    * @param config
+   * @param skipVerifyChecksum
    * @throws IOException
    */
   public static void parseKVContainerData(KeyValueContainerData kvContainerData,
-      ConfigurationSource config) throws IOException {
+      ConfigurationSource config, boolean skipVerifyChecksum) throws IOException {
 
     long containerID = kvContainerData.getContainerID();
 
     // Verify Checksum
-    ContainerUtils.verifyChecksum(kvContainerData, config);
+    // skip verify checksum if the state has changed to RECOVERING
```

Comment:
skip verify checksum if the state has changed to RECOVERING - > 

skip verify checksum if the state has changed to RECOVERING during container import.